docs/settings.md

```diff
@@ -633,3 +635,142 @@ Cookie: sessionid=xxxx
 
 To use this endpoint, an application needs a valid token, but no explicit
 permission is required.
+
+## Feature flags
+
+A feature flag is a name and an associated value (boolean, number, string or a
+JSON) that can be interpreted by the apps. It can be used for giving access to
+paid features, or to enable progressively a feature on all the cozy instances
+of a context. The stack computes the feature flags from several sources (in
+order of decreasing priority):
+
+- the flags set on the instance by the command `cozy-stack feature flags`
+- the response of the manager for `GET /api/v1/flags?sets=s1,s2&context=ctx`
+  (where `s1,s2` has been set via `cozy-stack feature sets`)
+- the flags coming from the context configuration (`context.<name>.features`)
+- the flags set on the context by the command `cozy-stack feature ratio`
+- the default value (`cozy-stack feature default`).
+
+For a given flag, the stack takes the value from the source with the highest
+priority.
```

Comment:
Please explicitely state here, in case of a json type flag, if they are merged or remplaced by higher-priority value. I guess replaced but it's better if it's explicit.

Reply:
I thought it was explicit: it takes the value from the source with the highest priority, and doesn't look at the other sources (no merge).